.github/actions/checkDeployBlockers/checkDeployBlockers.js

```diff
@@ -0,0 +1,46 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const {GITHUB_OWNER, EXPENSIFY_CASH_REPO} = require('../../libs/GithubUtils');
+
+const run = function () {
+    const octokit = github.getOctokit(core.getInput('GITHUB_TOKEN', {required: true}));
+    const issueNumber = Number(core.getInput('ISSUE_NUMBER', {required: true}));
+    let hasDeployBlockers = false;
+
+    return octokit.rest.issues.get({
```

Comment:
```suggestion
    return octokit.issues.get({
```

I've seen `rest` in the octokit docs, but none of our other actions have included that and it's all working as expected, so I _think_ that `getOctokit` returns a rest client (for the v3 GH API) rather than a GraphQL one